benchmark/benchmark.go

```diff
@@ -104,6 +108,36 @@ func (s *testServer) StreamingCall(stream testpb.BenchmarkService_StreamingCallS
 	}
 }
 
+func (s *testServer) UnconstrainedStreamingCall(stream testpb.BenchmarkService_UnconstrainedStreamingCallServer) error {
+	in := new(testpb.SimpleRequest)
+	go func() {
+		for {
+			if err := stream.(grpc.ServerStream).RecvMsg(in); err != nil {
+				return
```

Comment:
log.Error() (unless io.EOF)?

Reply:
err is expected to be Cancelled. So logging non-EOF errors clutters the consoles.

Reply:
log.Error() unless Cancelled then?